README.md

```diff
@@ -854,12 +830,11 @@ GIT_DEPTH: 0
 
 > You may be interested in using GitGuardian's [GitHub integration](https://dashboard.gitguardian.com/settings/workspace/integrations/github) to ensure full coverage of your GitHub projects as well as full git history scans and reporting.
 
-**ggshield's** support of GitHub comes in the form of GitHub actions.
+ggshield support for GitHub comes in the form of GitHub actions.
```

Comment:
this needs to be also corrected in public doc

Reply:
The public doc for GitHub action is a bit different than this one. I think it looks fine: https://docs.gitguardian.com/internal-repositories-monitoring/integrations/ci_cd_integrations/github_actions.